src/Versionable.php

```diff
@@ -23,6 +23,18 @@ trait Versionable
 
     public static function bootVersionable()
     {
+        if (config('versionable.keep_original_version')) {
+            static::updating(
+                function (Model $model) {
+                    if ($model->versions()->count() === 0) {
+                        $existingModel = self::find($model->id);
```

Comment:
`$model->id` result can be null and it will cause error when call `$existingModel->only(...)` due to $existingModel result is null

After look around I found that will happen when use custom primary key in model

Reply:
@jump-style Thanks for reviewing. This was fixed after the merge, we are now using $model->getKey()